packages/itwin/reports-config-widget/src/widget/components/BulkExtractor.ts

```diff
@@ -21,23 +21,30 @@ export default class BulkExtractor {
   private _extractionClientApi: ExtractionClient;
   private _accessToken: () => Promise<string>;
   private _reportStates = new Map<string, ExtractionStates>();
+  private _iModelStates = new Map<string, ExtractionStates>();
   private _timeFetched = new Date();
-  private _reportIds: string[];
-
-  constructor(apiConfig: ReportsApiConfig, reportIds: string[]) {
+  private _iModelRun = new Map<string, string>();
+  private _iModelToast = new Map<string, boolean>();
+  private _successfulExtractionToast: (iModelName: string, odataFeedUrl: string) => void;
+  private _failedExtractionToast: (iModelName: string) => void;
+
+  constructor(apiConfig: ReportsApiConfig,
```

Comment:
Move `apiConfig` to new line and the ctor args closing parenthesis to a new line. Now it is hard to see where the ctor args end.

Reply:
`apiConfig` moved to a new line